### ISSUE TITLE
Issues with removing packages from LocalRepo when signing packages

### ISSUE THREAD

**Issue by MithicSpirit** [Morganamilo/paru#495]
### Affected Version
`paru v1.8.0.r12.gba15ae4 - libalpm v13.0.0`

### Description
If `LocalRepo` and `Sign` are both enabled, then running `paru -Ld <locarepo-package>` will remove the package itself from the repo but not the `.sig` file; I believe this may also be related to an issue where attempting to reinstall `<aur-package>` after running `paru  -Ldd <aur-package>` will fail stating that the signature is invalid, though I cannot reproduce this consistently enough to test. The other issue is that `paru -Ld` produces extremely verbose output, listing all `.sig` files and stating that they are an "Unrecognized archive format" (output from this shown below).

### Output
```sh
$ paru -Ld digestif
==> Extracting aur.db.tar.gz to a temporary location...
==> Extracting aur.files.tar.gz to a temporary location...
==> Searching for package 'digestif'...
  -> Removing existing entry 'digestif-0.3-3'...
==> Creating updated database file '/var/lib/repo/aur/aur.db.tar.gz'
==> Signing database 'aur.db.tar.gz'...
  -> Created signature file 'aur.db.tar.gz.sig'
==> Signing database 'aur.files.tar.gz'...
  -> Created signature file 'aur.files.tar.gz.sig'
error: could not open file /var/lib/repo/aur/python-cloup-0.11.0-1-any.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/heroic-games-launcher-bin-1.9.1-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/xournalpp-git-nightly.r4.gb24535541-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/aur.db.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/xournalpp-git-1.1.0.r39.g4f8240210-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/dust-0.6.1-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/git-delta-git-r.1476.c733031-1-any.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/hsa-rocr-4.3.0-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/aur.db.tar.gz.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/protonfixes-git-1.0.15.r170.g060ceaa-1-any.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/bitwarden-1.27.1-2-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/digestif-0.3-3-any.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/rocm-device-libs-4.3.0-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/path-of-building-community-git-2.7.0.r4721.59.517-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/authy-1.8.4-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/mailspring-1.9.1-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/protontricks-1.6.0-1-any.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/zoom-5.7.4-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/manim-0.9.0-1-any.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/nyxt-browser-git-2.1.1.r470.g66c2b786-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/bottom-git-0.6.3.1199.g2dddfcc-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/audiotube-git-v0.1.r84.g1b29a02-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/mu-1.6.2-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/gping-git-gping.v1.2.3.r3.g93e7c8f-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/proton-ge-custom-bin-6.14_GE_2-0-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/python-lsp-server-1.2.0-1-any.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/signal-desktop-beta-bin-5.13.0beta.2-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/python-grpcio-tools-1.39.0-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/deadbeef-git-r10094.db3519efd-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/nvm-0.38.0-2-any.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/hyperfine-git-1.11.0.r67.g34558bf-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/comgr-4.3.0-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/aur.db.tar.gz.old.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/python-tifffile-2021.8.8-1-any.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/xmrig-6.14.0-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/brave-nightly-bin-1.30.8-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/flameshot-git-r1406.2f087980-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/piper-git-0.6.r1.g4dd9e4f-1-any.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/hsakmt-roct-4.3.0-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/python-ytmusicapi-0.19.0-1-any.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/linux-xanmod-headers-5.13.9-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/libratbag-git-0.16.r15.62f26e7-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/paru-git-1.8.0.r12.gba15ae4-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/llvm-amdgpu-4.3.0-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/pijul-1.0.0_alpha.53-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/rocm-cmake-4.3.0-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/pepper-flash-32.0.0.465-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/picom-git-1621_Next.59.g78e8666_2021.08.04-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: missing package metadata in /var/lib/repo/aur/aur.files
error: missing package metadata in /var/lib/repo/aur/aur.files.tar.gz
error: could not open file /var/lib/repo/aur/aur.files.tar.gz.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/i3lock-color-2.13.c.4-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/flashplayer-standalone-32.0.0.465-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/r-rcpp-1.0.7-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/emacs-pgtk-native-comp-git-28.0.50.149662-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/starship-git-0.56.0.1682.gb2e2330cb-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/r-rinside-0.2.16-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/linux-xanmod-5.13.9-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/yaml-language-server-bin-0.22.0-1-any.pkg.tar.zst.sig: Unrecognized archive format
error: missing package metadata in /var/lib/repo/aur/aur.db
error: could not open file /var/lib/repo/aur/rocm-opencl-runtime-4.3.0-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/vscodium-git-1.59.0.r0.gb1970e8-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/librewolf-bin-90.0.2-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: missing package metadata in /var/lib/repo/aur/aur.db.tar.gz
error: could not open file /var/lib/repo/aur/aur.files.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/goverlay-git-0.6.1.r0.g8cfb51e-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/czkawka-git-3.2.0.r0.g243505b-1-any.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/aur.files.tar.gz.old.sig: Unrecognized archive format
error: missing package metadata in /var/lib/repo/aur/aur.db.tar.gz.old
error: could not open file /var/lib/repo/aur/dunst-git-1.6.1.r95.g6d41d1c-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/digestif-git-0.3.157.3a9076f-1-any.pkg.tar.zst.sig: Unrecognized archive format
error: missing package metadata in /var/lib/repo/aur/aur.files.tar.gz.old
error: could not open file /var/lib/repo/aur/rocclr-4.3.0-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/popsicle-git-1.3.0.r43.g1a94779-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/szyszka-2.0.0-1-x86_64.pkg.tar.zst.sig: Unrecognized archive format
error: could not open file /var/lib/repo/aur/autotrash-0.4.4-1-any.pkg.tar.zst.sig: Unrecognized archive format
:: syncing local databases...
  syncing aur.db...
  syncing aur.db.sig...
```

paru.conf
```sh
#
# $PARU_CONF
# /etc/paru.conf
# ~/.config/paru/paru.conf
#
# See the paru.conf(5) manpage for options

#
# GENERAL OPTIONS
#
[options]
PgpFetch
Devel
Provides
DevelSuffixes = -git -cvs -svn -bzr -darcs -always -hg
#AurOnly
BottomUp
#RemoveMake
SudoLoop
UseAsk
#CombinedUpgrade
CleanAfter
UpgradeMenu
NewsOnUpgrade

NoWarn = dmenu-mithic-git

LocalRepo
Chroot
Sign = FEA8F724F2800E40
SignDb = FEA8F724F2800E40

#
# Binary OPTIONS
#
[bin]
FileManager = ranger
Pager = bat
#MFlags = --skippgpcheck
#Sudo = doas

[env]
_microarchitecture=99
use_numa=n
use_tracers=n
```

pacman.conf
```sh
#
# /etc/pacman.conf
#
# See the pacman.conf(5) manpage for option and repository directives

#
# GENERAL OPTIONS
#
[options]
# The following paths are commented out with their default values listed.
# If you wish to use different paths, uncomment and update the paths.
#RootDir     = /
#DBPath      = /var/lib/pacman/
#CacheDir    = /var/cache/pacman/pkg/
#LogFile     = /var/log/pacman.log
#GPGDir      = /etc/pacman.d/gnupg/
#HookDir     = /etc/pacman.d/hooks/
HoldPkg     = pacman glibc
#XferCommand = /usr/bin/curl -L -C - -f -o %o %u
#XferCommand = /usr/bin/wget --passive-ftp -c -O %o %u
CleanMethod = KeepInstalled KeepCurrent
Architecture = auto

# Pacman won't upgrade packages listed in IgnorePkg and members of IgnoreGroup
#IgnorePkg   =
#IgnoreGroup =

#NoUpgrade   =
#NoExtract   =

# Misc options
#UseSyslog
Color
#TotalDownload
CheckSpace
VerbosePkgLists
ParallelDownloads = 10
# default: 5
ILoveCandy

# By default, pacman accepts packages signed by keys that its local keyring
# trusts (see pacman-key and its man page), as well as unsigned packages.
SigLevel    = Required DatabaseOptional
LocalFileSigLevel = Optional
#RemoteFileSigLevel = Required

# NOTE: You must run `pacman-key --init` before first using pacman; the local
# keyring can then be populated with the keys of all official Arch Linux
# packagers with `pacman-key --populate archlinux`.

#
# REPOSITORIES
#   - can be defined here or included from another file
#   - pacman will search repositories in the order defined here
#   - local/custom mirrors can be added here or in separate files
#   - repositories listed first will take precedence when packages
#     have identical names, regardless of version number
#   - URLs will have $repo replaced by the name of the current repo
#   - URLs will have $arch replaced by the name of the architecture
#
# Repository entries are of the format:
#       [repo-name]
#       Server = ServerName
#       Include = IncludePath
#
# The header [repo-name] is crucial - it must be present and
# uncommented to enable the repo.
#

# The testing repositories are disabled by default. To enable, uncomment the
# repo name header and Include lines. You can add preferred servers immediately
# after the header, and they will be used before the default mirrors.

#[testing]
#Include = /etc/pacman.d/mirrorlist

[core]
Include = /etc/pacman.d/mirrorlist

[extra]
Include = /etc/pacman.d/mirrorlist

#[community-testing]
#Include = /etc/pacman.d/mirrorlist

[community]
Include = /etc/pacman.d/mirrorlist

# If you want to run 32 bit applications on your x86_64 system,
# enable the multilib repositories as required here.

#[multilib-testing]
#Include = /etc/pacman.d/mirrorlist

[multilib]
Include = /etc/pacman.d/mirrorlist

# An example of a custom package repository.  See the pacman manpage for
# tips on creating your own repositories.
#[custom]
#SigLevel = Optional TrustAll
#Server = file:///home/custompkgs

[aur]
SigLevel = PackageOptional DatabaseOptional
Server = file:///var/lib/repo/aur
```

**Comment by MithicSpirit** [comment]
Now that I think about it I think that the last problem (about the verbose output) may be with the `repo-remove` script instead. If so, please let me know and I can try to submit it as an issue there instead. (I believe the first issue can still be fixed in `paru` so that is why I am leaving this issue open)

**Comment by Morganamilo** [comment]
Fixed in 2a1a412 and f4b9655.
